pkg/apis/rke.cattle.io/v1/zz_generated_deepcopy.go

```diff
@@ -1,4 +1,3 @@
-//go:build !ignore_autogenerated
```

Comment:
Not sure if this should be included.

Reply:
it knocks go generate back in sync not sure where it was missed from.